nxc/modules/get-unixUserPassword.py

```diff
@@ -0,0 +1,63 @@
+from impacket.ldap import ldapasn1 as ldapasn1_impacket
+from impacket.ldap import ldap as ldap_impacket
+from nxc.logger import nxc_logger
+
+
+class NXCModule:
+    """
+    Get unixUserPassword attribute attribute stored in ldap.
```

Comment:
"attribute" is double. And it would be nice to know that it retrieves the attribute from all users, maybe something like "Get unixUserPassword attribute from all users in ldap"